extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -29,8 +30,7 @@
 			this.previewFrame = $('#PreviewFrame');
 
 			// min width for dynamic is equal to our breakpoint
-			// TODO: When refactoring ThemeDesigner into AMD module, use wikia.fluidlayout.getBreakpointContent
-			this.minWidthForDynamicBackground = 1030;
+			this.minWidthForDynamicBackground = fluidlayout.getBreakpointContent();
 			// TODO: when using wikia.fluidlayout for minWidthForDynamicBackground, we need to add following variable to
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)